plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/TestHive.java

```diff
@@ -60,6 +60,20 @@ private int getHiveVersionMajor()
         return hiveVersionMajor;
     }
 
+    @Test
+    public void testInsertBucketedTransactionalTableLayout()
```

Comment:
I missed the explanation why this cant be in `AbstractTestHive` while `insertBucketedTableLayout(false)` and `insertPartitionedBucketedTableLayout(false)` are

Reply:
Because if they are in `AbstractTestHive` then  `TestHiveAlluxioMetastore` fails because of them.

Reply:
> Because if they are in AbstractTestHive then TestHiveAlluxioMetastore fails because of them.

Add them to `AbstractTestHive` and override them in ` TestHiveAlluxioMetastore` with a comment

Reply:
Ok I will do that, but I am not sure why this is a better way.

Reply:
> Ok I will do that, but I am not sure why this is a better way.

We use this approach in other `abstract/dervied` tests